docs/Documentation/Visual-Studio-extensions-and-snippets.markdown

```diff
@@ -0,0 +1,15 @@
+## Visual Studio Extensions and Snippets
+
+There are already a few Visual Studio extensions and snippets ready to help your everyday Orchard development work.
+
+### Visual Studio Extensions
+
+Orchard-related extensions are listed here in alphabetical order:
+
+  * **Lombiq Orchard Visual Studio Extension**: Visual Studio extension with many features and templates frequently used by the [Lombiq](http://lombiq.com) developers. Contains Orchard-related as well as generic goodies. Repository: [https://github.com/Lombiq/Lombiq-Orchard-Visual-Studio-Extension](https://github.com/Lombiq/Lombiq-Orchard-Visual-Studio-Extension).
+  * **Orchardizer**: an extension for Visual Studio 2013 by [Harry Westbrook](http://arkleseizure.net) that reduces development of Orchard modules to a bunch of textboxes and some sporadic clicks. Repository: [https://github.com/Hazzamanic/orchardizer](https://github.com/Hazzamanic/orchardizer).
+  * **VS Constructor Field Refactoring rule**: an extension by [Sebastien Ros](http://sebastienros.com/) to generate a private readonly field based on a constructor parameter. Repository: [https://github.com/sebastienros/vsconstructorfield](https://github.com/sebastienros/vsconstructorfield).
```

Comment:
Small point, his name is Sébastien with the squiggle. I think the anglicised version is acceptable but looking at his social profiles he does use it consistently that way.